scalding-core/src/test/scala/com/twitter/scalding/mathematics/Matrix2Test.scala

```diff
@@ -40,6 +44,37 @@ class Matrix2Sum(args: Args) extends Job(args) {
   sum.write(TypedText.tsv[(Int, Int, Double)]("sum"))
 }
 
+class Matrix2SumOrderedSerialization(args: Args) extends Job(args) {
+
+  import Matrix2._
+  import cascading.pipe.Pipe
+  import cascading.tuple.Fields
+  import com.twitter.scalding.TDsl._
+
+  override def config = super.config + (Config.ScaldingRequireOrderedSerialization -> "true")
+  implicit val intOS: OrderedSerialization[Int] = new OrderedSerialization[Int] {
```

Comment:
can you add a comment that this is just for a test, normally you would use: 
https://github.com/twitter/scalding/blob/develop/scalding-core/src/main/scala/com/twitter/scalding/serialization/RequiredBinaryComparators.scala#L21

```scala
import RequiredBinaryComparators.orderedSerialization
```